locks_test.go

```diff
@@ -16,7 +16,7 @@ func (s *LimitersTestSuite) useLock(lock limiters.DistLocker, shared *int, sleep
 	// Check for the race condition.
 	s.Equal(sh, *shared)
 	*shared++
-	s.NoError(lock.Unlock())
+	s.NoError(lock.Unlock(context.TODO()))
```

Comment:
According to the docs `context.Background()` should be used in tests.
https://pkg.go.dev/context#Background